PublicationsRouter/v2/sword-out/XWALK-DSpace.md

```diff
@@ -0,0 +1,115 @@
+# JPER Core Metadata to Dublin Core for DSPACE
+
+This document describes the crosswalk from the notification metadata fields received via the JPER API to the
+ XML format supplied to repositories via SWORDv2.
+ 
+For information about the schema see:
+
+* [Dubin Core](http://dublincore.org/documents/dcmi-terms/)
+
+The following table lists the notification fields in the JSON, then the DC terms fields it will be transformed to, and
+any notes regarding the transformation:
+
```

Comment:
The following table lists the fields from the JPER API JSON notification, then the corresponding DC fields it will be transformed to, and any notes regarding the transformation: 

Think this reads clearer.